blank-canvas-blocks/sass/blocks/_file.scss

```diff
@@ -0,0 +1,17 @@
+// TODO: Remove when https://github.com/WordPress/gutenberg/issues/27760 is fixed
+.wp-block-file {
+	// Undo Gutenberg hover defaults
+	a.wp-block-file__button:active,
+	a.wp-block-file__button:focus,
+	a.wp-block-file__button:hover,
+	a.wp-block-file__button:visited {
+		color: var(--wp--custom--color--background);
+		background: var(--wp--custom--color--secondary);
```

Comment:
Would it be unreasonable to return the `custom.button.color.text` and `custom.button.color.background` values to theme.json?  The `core/button` style could be mapped to that variable (the right way) and the things like this (and search block) could use the same color variable.

In this way changing the color value of `custom.button.color.text/.background` changes the colors of all the buttons, and the choice to leverage `custom.color.primary` or `custom.color.secondary` as the button color is at the discretion of the configuration of theme.json instead of a decision to be made in the polyfill.

Reply:
This makes sense to me too